doc/user-guide.md

```diff
@@ -671,7 +671,7 @@ deleted until you remove the finalizer (ie, after your cleanup logic has success
 
 **Example:**
 
-The following is a snippet from the controller file under `pkg/controller/memcached/memcached_controller.go`
+The following is a snippet from the controller file under `test/test-framework/pkg/controller/memcached/memcached_controller.go`
```

Comment:
The original path is based on the example, I mean in the tests performed by the user and then IMO we should not change it for the place where we stored the sample.

Reply:
Yes, @camilamacedo86 , my wrong, I will change in the example repo after release.